src/dali/abstract_pair_converter_plugin.py

```diff
@@ -179,35 +224,28 @@ def _build_fiat_list(self) -> None:
             LOGGER.info("Fetching of fiat symbols failed. The server might be down. Please try again later.")
             raise RP2RuntimeError("JSON decode error") from exc
 
-    def _add_fiat_edges_to_graph(self, graph: Dict[str, Dict[str, None]], markets: Dict[str, List[str]]) -> None:
+    def _add_fiat_edges_to_graph(self, graph: MappedGraph[str], markets: Dict[str, List[str]]) -> None:
         if not self.__fiat_list:
             self._build_fiat_list()
 
         for fiat in self.__fiat_list:
             to_fiat_list: Dict[str, None] = dict.fromkeys(self.__fiat_list.copy())
             del to_fiat_list[fiat]
-            if graph.get(fiat):
+            # We don't want to add a fiat vertex here because that would allow a double hop on fiat (eg. USD -> KRW -> JPY)
```

Comment:
Just curious: what's wrong with a double fiat hop?

Reply:
Well, first of all, our current data provider for fiat pricing allows us to convert from any fiat to any other fiat, so it is unnecessary. But, on top of that, we only currently have daily exchange rates and so I felt like we need to prioritize the more granular data (crypto exchange rates) for now.

In the future, if we have a different provider, we may need to double hop, but we will need a different graph-building function at that point, so we can revisit it then.